components/burger/Burger.module.css

```diff
@@ -0,0 +1,38 @@
+.burger {
+  position: absolute;
+  top: 4%;
+  right: 4%;
```

Comment:
I would probably use rem here, as this translates to pixels and the percentage is actually dependent on the device width or height.

Reply:
Thx!